### ISSUE TITLE
Cockatrice is unable to load the card database

### ISSUE THREAD

**Issue by TheAller** [Cockatrice/Cockatrice#3309]
<b>System Information:</b>
Build Architecture: 64-bit
Qt Version: 5.11.0
System Locale: it_CH

macOS Sierra (10.12.6)
__________________________________________________________________________________________

Hi, I've been trying to install Cockatrice on a second computer I own and there seems to be no way to make it work. 

**As soon as I download Cockatrice and open it, I get this message:** 

"Cockatrice is unable to load the card database.
Do you want to update your card database now?
If unsure or first time user, choose "Yes"

**If I choose "Yes", I get the following message:**

Unable to run the card database updater: /private/var/folders/74/vn4gzszd78sdl8njw8xf18d00000gn/T/AppTranslocation/E0302CA5-33AA-43A8-85AF-9D3499215858/d/oracle 

I tried to uninstall and download again Cockatrice several times and it's always the same.
I looked at several older posts on this forum that seemed to report similar problems, and tried the solutions that were written there. 

Firstly, I made sure I actually dragged both the Cockatrice icon and the Oracle icon in the Applications folder after download. It didn't matter.

Secondly, I tried to download manually the "AllSets.json.zip" file and changing the preferences on Cockatrice in order to load the card database from a local file. **Howewer, when doing so, I get the following message:**

"File Error loading your card database. Would you like to change your database location setting?"

**When I close the "preferences" tab of Cockatrice, I get the following message:** 

"Your card database is invalid.
Cockatrice may not function correctly with an invalid database
You may need to rerun oracle to update your card database.
Would you like to change your database location setting?"

**If I go to "check for card updates", I get the following message:** 

"A card database update is already running."

I have no clue what to do about this and was wondering if anyone on here could help.

**Comment by ctrlaltca** [comment]
Please have a look at the comment https://github.com/Cockatrice/Cockatrice/issues/3148#issuecomment-376231038 , the issue is similar and maybe it can help

**Comment by Daenyth** [comment]
tldr; You need to install cockatrice to /Applications, not run it from inside the DMG

**Comment by TheAller** [comment]
I forgot to mention I had seen that post too.

I made sure to copy Cockatrice and Oracle in the applications folder and open them from there. 

I just deleted once more the files an re-downloaded them, copied them in the applications folder, double click on the icon in the applications folder: it still doesn't work.

**Comment by Daenyth** [comment]
The error message you pasted includes a DMG-mounted folder. Ensure you've ejected the DMG and are launching via /Applications.

**Comment by TheAller** [comment]
But what can I do other than opening the Applications folder and double click on the icon there? I even deleted the DMG folder from my computer. It's still the same.

**Comment by TheAller** [comment]
Well...actually there WAS something I hadn't tried. And tha was...restarting my computer. It seems like downloading the files 1'000 times and reading a bunch of old threads all I had to do was restart my computer. Sorry for wasting your time.

**Comment by ctrlaltca** [comment]
Looks like old school solutions always work!

![it](https://user-images.githubusercontent.com/1631111/41917313-63017cc0-795a-11e8-8612-0a1a3bd9c687.jpg)

**Comment by CookieMonster214** [comment]
> Well...actually there WAS something I hadn't tried. And that was...restarting my computer. It seems like instead of downloading the files 1'000 times and reading a bunch of old threads all I had to do was restart my computer. Sorry for wasting your time.

Well, I did the same thing and it did not help.
